README.md

```diff
@@ -7,6 +7,76 @@
 Retrieve scanned documents along with information extracted with OCR engine. Store the images and let recipient
 services fetch the new data.
 
+## Building and deploying the application
+
+The project uses [Gradle](https://gradle.org) as a build tool. It already contains
+`./gradlew` wrapper script, so there's no need to install gradle.
+
+### Building the application
+
+To build the project execute the following command:
+
+```bash
+  ./gradlew build
+```
+
+### Running the application
+
+Create the image of the application by executing the following command:
+
+```bash
+  ./gradlew installDist
```

Comment:
I think `assemble` works better with spring boot 2. could be wrong and this one is ok too